torch/_dynamo/variables/user_defined.py

```diff
@@ -969,48 +964,6 @@ def call_function(
             obj_src = AttrSource(self.source, "__self__")
             obj_var = VariableTracker.build(tx, obj, obj_src)
             return func_var.call_function(tx, [obj_var] + args, kwargs)
-        elif (
-            istype(self.value, functools.partial)
-            and trace_rules.lookup(self.value.func)
-            == variables.TorchInGraphFunctionVariable
-            and all(
-                variables.ConstantVariable.is_literal(v)
-                for v in itertools.chain(self.value.args, self.value.keywords.values())
-            )
-        ):
```

Comment:
This branch won't happen anymore due to https://github.com/pytorch/pytorch/blob/3455ab44a75ab1fac45998e3f0be19eb7a136602/torch/_dynamo/variables/builder.py#L731-L756